apps/src/lab2/views/components/TextToSpeech.tsx

```diff
@@ -14,28 +13,26 @@ interface TextToSpeechProps {
  * TextToSpeech play button.
  */
 const TextToSpeech: React.FunctionComponent<TextToSpeechProps> = ({text}) => {
+  const {isTtsAvailable, speak} = useBrowserTextToSpeech();
+
   const playText = useCallback(() => {
-    const currentLocale = getCurrentLocale();
-    const voices = speechSynthesis.getVoices();
-    if (voices.length === 0) {
+    if (!isTtsAvailable) {
+      console.log('Browser TextToSpeech unavailable');
```

Comment:
I see below you have a todo for better UI for disabled state; should replacing this console log and the one in BrowserTextToSpeech be a part of some follow up item?

Reply:
Oh yeah good call; I'm gonna be cleaning up this UI shortly so I'll take care of it as part of that.